common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -115,8 +115,9 @@
   "feature.wynntils.chatCoordinates.name": "Chat Coordinates",
   "feature.wynntils.chatItem.description": "Adds the ability to share items in chat.",
   "feature.wynntils.chatItem.name": "Chat Item",
-  "feature.wynntils.chatMention.aliases.description": "Any aliases you wish to get mentioned for in addition to your username. Names should be separated with an single comma.",
+  "feature.wynntils.chatMention.aliases.description": "Any aliases you wish to get mentioned for in addition to your username. Names should be separated with an single comma. Special characters (except underscore, dash, and space) will be filtered out.",
   "feature.wynntils.chatMention.aliases.name": "Aliases",
+  "feature.wynntils.chatMention.aliasesParseFailed": "One or more aliases failed to parse. Aliases should be separated by a single comma. Special characters (except underscore, dash, and space) are not allowed.",
```

Comment:
This mentions underscore, but the pattern is `"[^\\w -]"`. Which one is incorrect?

Reply:
Neither, \\w is alphanumeric and underscore

Reply:
Oh. I see. I thought it was just A-Z. TIL. Thanks!